### PR TITLE
add  work and about section

### PR DESCRIPTION
**Pull request by Gedewon**
# in this pull request I added two basic components work and about me section 
### list of sections added :
- modifying index.html 
- add a section for the work
- add a section for the about section 
- - parsing the Figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,6 +82,225 @@ <h2>
...
+          <div class="headline-text">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <p class="let-connect">LET'S CONNECT</p>
```
[the remark this answers is not in the document]
Comment (Gedewon) [reply index.html]: @nicupop729 resolved

File: index.html [hunk 2]
```diff
@@ -52,7 +52,7 @@ <h2>
           </p>
         </div>
 
-        <p class="let-connect">LET'S CONNECT</p>
+        <h2 class="let-connect">LET'S CONNECT</h2>
```
Comment (codepantherr) [inline index.html:55]: - [ ] This appears too big and makes your UI look clunky. Please follow the specification in the Figma template. The `font-size` should be `16px` 👍

File: index.html [hunk 3]
```diff
@@ -82,6 +82,225 @@ <h2>
...
+          <div class="headline-text">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <h2 class="let-connect">LET'S CONNECT</h2>
```
Comment (codepantherr) [inline index.html:214]: - [ ] Please see my comment above about this ☝️ .

**Review by codepantherr — CHANGES_REQUESTED** [review]
Hi @Gedewon 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You have done well, but there are still a few changes you need to make to move forward.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Gedewon) [reply index.html]: @codepantherr resolved

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Gedewon ,  

Your project is complete!  it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gedewon** [comment]
this is my last project approved.so please check it I think you miss
understod me.I am not repeting the week okay.

**Pull request merged**